interfaces/builtin/locale_control.go

```diff
@@ -33,6 +33,40 @@ const localeControlBaseDeclarationSlots = `
 const localeControlConnectedPlugAppArmor = `
 # Description: Can manage locales directly separate from 'config ubuntu-core'.
 
+#include <abstractions/dbus-strict>
+
+# Introspection of org.freedesktop.locale1
+dbus (send)
+	bus=system
+	path=/org/freedesktop/locale1
+	interface=org.freedesktop.DBus.Introspectable
+	member=Introspect,
+# Properties of org.freedesktop.locale1
+dbus (send)
+	bus=system
+	path=/org/freedesktop/locale1
+	interface=org.freedesktop.DBus.Properties
+	member=Get{,All},
+# do not use peer=(label=unconfined) here since this is DBus activated
+dbus (send)
+	bus=system
+	path=/org/freedesktop/locale1
+	interface=org.freedesktop.locale1
+	member={SetLocale,SetX11Keyboard,SetVConsoleKeyboard}
+	peer=(label=unconfined),
+dbus (send)
+	bus=system
+	path=/org/freedesktop/locale1
+	interface=org.freedesktop.locale1
+	member={SetLocale,SetX11Keyboard,SetVConsoleKeyboard}
+	peer=(name=org.freedesktop.locale1),
```

Comment:
Is there value in the duplicated `peer=(label=unconfined)` instance of this rule? Since we want to support both a confined and unconfined version then I think it would be easier to just drop that version and just use the `peer=(name=org.freedesktop.locale1)` rule since that should capture both the unconfined and confined cases. In that case, adding a comment like the standard `do not use peer=(label=unconfined) here since this can be DBus activated` one would be worthwhile.

Reply:
You are right, it seems to work. Changing it.

Reply:
Fixed.